src/govuk/helpers/_typography.scss

```diff
@@ -106,12 +107,15 @@
 ///   )
 /// );
 ///
-/// @param {Map} $font-map - Font map
+/// @param {Number} $size - Point from the spacing scale (typically the size as
```

Comment:
You've written "**typically** the size..." where as below (lines 174 and 175) you've omitted the "typically". I'm wondering if we should either make these consistent or if we should be more specific here. I'm leaning towards the latter as my understanding is that the only instance when the `$size` wouldn't be a number on the font scale is if you wanted to pass `false` and not use the scale. This also contradicts the `@throw` on like 117.

Can you think of other scenarios besides a font scale value of `false`? If not, I'd suggest changing this and the similar line on 174 to something like:

>Point from the spacing scale (the size in px as it would appear on tablet and above) or `false`

I'd similarly either omit the `@throw` on line 117 or change it to:

>if `$size` is not a valid point from the spacing scale or `false`

Reply:
> Can you think of other scenarios besides a font scale value of false?

The other scenario where a point from the spacing scale wouldn't be the size on tablet and above is if the user has overridden `$govuk-typography-scale` – the fact that the 'key' is the same as the tablet size is only a convention and is not enforced in code, so a user could choose to use a different set of keys, e.g:

```scss
$govuk-typography-scale: (
    1: (
        null: (
        font-size: 53px,
        line-height: 55px
        ),
        tablet: (
        font-size: 80px,
        line-height: 80px
        ),
        print: (
        font-size: 53pt,
        line-height: 1.1
        )
    ),
    2: (
        null: (
        font-size: 32px,
        line-height: 35px
        ),
        tablet: (
        font-size: 48px,
        line-height: 50px
        ),
        print: (
        font-size: 32pt,
        line-height: 1.15
        )
    )
    // etc
)
```

However, this is a rather theoretical edge case and, because of the way we use the typography scale throughout the components, in reality would result in lots of things breaking.

> This also contradicts the @throw on like 117.

> I'd suggest changing this and the similar line on 174 to something like:
> 
>> Point from the spacing scale (the size in px as it would appear on tablet and above) or false

You can't call `govuk-typography-responsive` with `false` – it doesn't make sense as setting the font size is the only thing the mixin does, so it'll throw an error. You **can** call `govuk-font` with `$size: false` as it does lots of other things.